### PR TITLE
C1 zoheiry auto level

### PR DESCRIPTION
**Pull request by zoheiry**
issue #167 
@adamggg // Code Reviewer
@KareemAliRahman // Scenario Reviewer
@Khloud-Khalid // UML Reviewer
@mostafahassaan // Doc & testing Reviewer
Please Review everyone. Thank You

### CODE REVIEW

File: arability/app/assets/javascripts/game.js [hunk 1]
```diff
@@ -374,7 +381,12 @@ function initializeGame(){
 function initializeList(){
 	var lsHtml = [];
```
Comment (adamggg) [inline arability/app/assets/javascripts/game.js:381]: What does this variable represent?

File: arability/app/assets/javascripts/game.js [hunk 2]
```diff
@@ -653,6 +671,12 @@ function calculatePossible(){
 			$('#' + lsId).addClass('text-warning');
 			$('#' + lsId).css( "color", "orange" );
 		}
+		else{
+			var postfixNum = k - 1;
+			lsId = "ls" + postfixNum;
+			$('#' + lsId).removeClass('text-warning');
+			$('#' + lsId).css( "color", "#333333" );
```
Comment (adamggg) [inline arability/app/assets/javascripts/game.js]: Remove extra whitespaces

File: arability/app/assets/javascripts/game.js [hunk 3]
```diff
@@ -1006,6 +1033,36 @@ function startPulling(r, c, count){
 }
```
Comment (adamggg) [inline arability/app/assets/javascripts/game.js]: Remove this extra line too =)

File: arability/app/assets/javascripts/game.js (hunk 1 above)
Comment (zoheiry) [inline arability/app/assets/javascripts/game.js:381]: lsHTML, its List HTML, the HTML in the list, its pretty clear :)

File: arability/app/assets/javascripts/game.js (hunk 2 above)
Comment (zoheiry) [inline arability/app/assets/javascripts/game.js]: the white spaces in $('#' + lsId).css( "color", "#333333" ), or do you mean all white spaces including the ones before and after arithmetic operations ?

Comment (adamggg) [inline arability/app/assets/javascripts/game.js]: The one after the css( and before the closing)

AG

**Comment by zoheiry** [comment]
fixed issues, please review again @adamggg

File: arability/app/assets/javascripts/game.js (hunk 1 above)
Comment (adamggg) [inline arability/app/assets/javascripts/game.js:381]: Alright =) Does make sense ls := list in commands =D

**Pull request merged**